assets/js/components/BatchEdit/About/ControlledMetadata.jsx

```diff
@@ -39,10 +39,10 @@ const BatchEditAboutControlledMetadata = ({ ...restProps }) => {
 
   return (
     <div data-testid="controlled-metadata" {...restProps}>
-      <ul>
+      <ul className="columns is-multiline">
         {!codeLists.isLoading &&
           CONTROLLED_METADATA.map(({ label, name, scheme }) => (
-            <li key={name} className="mb-5" data-testid={name}>
+            <li key={name} className="column is-half mb-5" data-testid={name}>
```

Comment:
Can we make it two lines with less margin/padding as below always instead of space-permitting?

<img width="1449" alt="Screen Shot 2020-11-04 at 11 21 48 AM" src="https://user-images.githubusercontent.com/72217877/98146666-57610780-1e91-11eb-8440-a2e35571daa7.png">

Reply:
Yea, good call.   Those are buttons (text buttons, so not totally sure how it'd work but I'll see if there's something we can do.

Reply:
Thank you!